images/client/Containerfile.centos

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Michael Adam
+
+FROM quay.io/centos/centos:stream9
+
+MAINTAINER Michael Adam <obnox@samba.org>
+
+LABEL org.opencontainers.image.title="Samba Client container"
+LABEL org.opencontainers.image.description="Samba Client container"
+LABEL org.opencontainers.image.vendor="Samba in Kubernetes"
+
```

Comment:
Following the same LABELs we use in `samba-operrator`, please add also `org.opencontainers.image.url="https://github.com/samba-in-kubernetes/samba-container`